### ISSUE TITLE
Complete The Upgrading Guide

### ISSUE THREAD

**Issue by GrahamCampbell** [thephpleague/factory-muffin#362]
I need to complete the upgrading guide for 3.0.

**Comment by igorsantos07** [comment]
Please, take the site down or redirect to the README if there's not an update of the docs for around 4 years. It's quite deceptive, to say the very least, and I guess I wasn't the only one tricked :)

**Comment by GrahamCampbell** [comment]
The docs are already up to date for new comers. There are just some details missing for old users wanting to upgrade to the very newest version.

**Comment by GrahamCampbell** [comment]
RE the website, which is the topic of issue #448, I would accept PRs to address this. The website is open source and deployed from this repo itself (see the gh-pages* branches).

**Comment by igorsantos07** [comment]
Right, got the wrong issue. I was at #448 and got here because of that comment. I guess I felt on the same trap, quick-reading.

**Comment by igorsantos07** [comment]
Just wanted to point out that the upgrade guide, as well as the README, are missing information about how to customize how instance, save, and delete happens. I'm guessing I'll have to create a custom store for my case, which is also severely different from the old ways.

**Comment by GrahamCampbell** [comment]
I'd accept PRs to improve the docs. :)

**Comment by igorsantos07** [comment]
Hope you get some.
